### ISSUE TITLE
Mention `oca-port` in migration procedure wiki page

### ISSUE THREAD

**Issue by sbidoul** [OCA/maintainer-tools#589]
I'm under the impression that oca-port is not mature enough to recommend in the [migration procedure wiki page](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0).

What do people think?

@sebalix would you be able to propose that (while keeping the manual procedure in the document for now)?

**Comment by pedrobaeza** [comment]
In any case, it should be `odoo-module-migrator` instead.

**Comment by sbidoul** [comment]
Why "instead"? In my understanding they are complementary tools, with `oca-port` caring about the branch creation and cherry-picking, while `odoo-module-migrator` is to transform the source code. That's why I created a separate issue.

**Comment by pedrobaeza** [comment]
I think `odoo-module-migrator` does also the commit history fetching.

**Comment by sebalix** [comment]
Yes `odoo-module-migrator` does the `git format-patch` as well, that's the only common point I think, otherwise `oca-port` is only about playing with git repositories/commits, while `odoo-module-migrator` deals with code transformation in respect to the targeted Odoo version. In fact it could be called after `oca-port` process during a migration.

@sbidoul agree, `oca-port` is not mature enough, and we aim to change a bit the input parameters so I would not recommend this tool yet in the migration guide.

**Comment by sbidoul** [comment]
> agree, oca-port is not mature enough, and we aim to change a bit the input parameters so I would not recommend this tool yet in the migration guide.

Ow, I had mistyped "not" where I meant "now mature" enough. But, good to now there are still changes upcoming in the CLI. Thanks for the update.

**Comment by sebalix** [comment]
That said, `odoo-module-migrator` exists since a long time so it deserves to be mentioned in the doc.
Once I'm confident with `oca-port` stability, and if it doesn't break any workflow with other tools I'll be glad to mention it as well.
